.pre-commit-config.yaml

```diff
@@ -17,3 +17,8 @@ repos:
     hooks:
       - id: black
         language_version: python3.8
+-   repo: https://github.com/timothycrosley/isort
+    rev: 5.6.4
+    hooks:
+    -   id: isort
+        args: ["--profile", "black"]
```

Comment:
@david26694 I copied this from ds analytics

Reply:
Should I run isort in the entire repo in another pr?

Reply:
yeah pls